test/config.go

```diff
@@ -22,6 +22,7 @@ type Config struct {
 	ManagerTag   string
 	DaemonImage  string
 	DaemonTag    string
+	TimerImage   string
```

Comment:
I tend to name it as E2eImage as we would put more binary to help e2e test in this image, not only timechaos.

Reply:
You mean we may add some other binaries to this image? Make sense!

Will update it tomorrow